### ISSUE TITLE
Call to an undefined method Illuminate\Contracts\Routing\ResponseFactory|Symfony\Component\HttpFoundation\Response::json()

### ISSUE THREAD

**Issue by canvural** [nunomaduro/larastan#67]
Using larastan 0.1.8

This code:
```
\response()->json();
```

gives

```
Call to an undefined method Illuminate\Contracts\Routing\ResponseFactory|Symfony\Component\HttpFoundation\Response::json()
```

on 8th level.

**Comment by lasselehtinen** [comment]
Same thing happens with all other response helper methods like make, json, jsonp, view, header etc.

```
return response('foobar')->header('Content-Type', 'text/csv');

Call to an undefined method Illuminate\Contracts\Routing\ResponseFactory|Symfony\Component\HttpFoundation\Response::header()
```

**Comment by nunomaduro** [comment]
@canvural  This issue got addressed on the version `v0.2.11`. Can you test it?

**Comment by canvural** [comment]
@nunomaduro Yep, its fixed. Thank you!
